NAMESPACE

```diff
@@ -24,5 +24,3 @@ export(dccTab)
 export(dccTabs)
 export(dccTextarea)
 export(dccUpload)
-export(dccTimer)
-export(dccGeolocation)
```

Comment:
I believe so. I'm not sure why those are in the `NAMESPACE` to begin with since they're not included in the `dev` branch yet, but I think it's safe to remove them for now until they're added back with the merge of that PR.